pipeline/plot/unit_characteristic_plot.py

```diff
@@ -306,6 +306,86 @@ def plot_unit_bilateral_photostim_effect(probe_insertion, clustering_method=None
     return fig
 
 
+def plot_driftmap(probe_insertion, clustering_method=None, shank_no=1):
```

Comment:
does this one need docs too?
(can defer this on worker components implementation as noted below if that makes more sense)

Reply:
yes will need docs for this function as well as the corresponding `report` table, not yet implemented